src/components/CandidatesTab.tsx

```diff
@@ -0,0 +1,55 @@
+import {
+  ElectionAPI,
+  ElectionBallotMeta,
+  ElectionCandidatesTableSection,
+  ElectionScopeIncomplete,
+  DivBody,
+  useApiResponse,
+} from "@code4ro/reusable-components";
+import React from "react";
+import { Separator } from "./Separator";
+import { EmbedButton, EmbedButtonWrapper } from "./EmbedButton";
+import classes from "./CandidatesTab.module.scss";
+
+type Props = {
+  api: ElectionAPI;
+  meta?: any;
+  ballot?: any;
+  scope: any;
+  onScopeChange?: (scope: ElectionScopeIncomplete) => unknown;
+  loading?: boolean;
+};
+
+export const CandidatesTab: React.FC<Props> = ({ api, meta, scope }) => {
+  const { ballot: ballotName, ballotId } = meta || {};
+  const { countyId } = scope || {};
+
+  const { data } = useApiResponse(() => {
+    return {
+      invocation: (api && ballotId != null && api.getCandidates(ballotId, 2, countyId)) || undefined,
+      discardPreviousData: true,
+    };
+  }, [api, ballotId, scope.type, scope.countyId]);
+
+  return (
+    <EmbedButtonWrapper>
+      {meta && meta.live && (
+        <>
+          <DivBody className={classes.processHeading}>
+            Graficele arată procentele obținute de fiecare candidat pe măsură ce sunt numărate voturile. Datele se
```

Comment:
This is not needed for the candidates tab.

Reply:
👍 removed